.env.sample

```diff
@@ -19,3 +19,6 @@ SESSION_COOKIE_SECRET=moncompteprosecret
 # INSEE credentials needed to fetch organization info
 INSEE_CONSUMER_KEY=GET_YOUR_OWN
 INSEE_CONSUMER_SECRET=GET_YOUR_OWN
+# url where legal pages are visible to the public
+EXTERNAL_CONTENT_URL=https://legal.moncomptepro.beta.gouv.fr
+AUTOUPDATE_EXTERNAL_CONTENT=True
```

Comment:
depending on what we decide how this behaves, this option can be removed, it's more there as an example…

Reply:
If these variables has only one value accross all env, they should not be env var IMHO.